signer/awsv2/sdkv2signer.go

```diff
@@ -68,6 +68,7 @@ func (s *awsSdkV2Signer) SignRequest(r *http.Request) error {
 	}
 
 	hash, err := hexEncodedSha256OfRequest(r)
+	r.Header.Set("X-Amz-Content-Sha256", hash)
```

Comment:
Shall we move this to constant?

Reply:
Since its just used here, I kept it as a string